lib/linguist/heuristics.rb

```diff
@@ -392,6 +392,26 @@ def call(data)
       end
     end
 
+    disambiguate ".properties" do |data|
+      if /^[^#!;][^=]*=/.match(data)
+        # Match all key=value, if no comment token
+        if /^[;\[]/.match(data)
+          # Match INI comment token or sections
+          Language["INI"]
+        elsif /^[#!]/.match(data)
+          # Match Java comment tokens
+          Language["Java Properties"]
+        else
+          # Assume key=value is INI
+          Language["INI"]
+        end
+      end
+      if /^[^#!][^:]*:/.match(data)
```

Comment:
Should this be an `elsif`? It's either `key=value` or `key:value`, right? I think that will fix the test failure.